definitions/features/pulpcore.rb

```diff
@@ -3,10 +3,34 @@
 class Features::Pulpcore < ForemanMaintain::Feature
   include ForemanMaintain::Concerns::PulpCommon
 
+  TIMEOUT_FOR_TASKS_STATUS = 300
+  RETRY_INTERVAL_FOR_TASKS_STATE = 10
+
   metadata do
     label :pulpcore
   end
 
+  def cli(args)
+    parse_json(execute("pulp --format json #{args}"))
+  end
+
+  def running_tasks
+    cli('task list --state-in running --state-in canceling')
+  end
+
+  def wait_for_tasks(spinner, timeout_for_tasks_status = TIMEOUT_FOR_TASKS_STATUS)
+    Timeout.timeout(timeout_for_tasks_status) do
+      while (task_count = running_tasks.length) != 0
+        puts "\nThere are #{task_count} tasks."
+        spinner.update "Waiting #{RETRY_INTERVAL_FOR_TASKS_STATE} seconds before retry."
+        sleep RETRY_INTERVAL_FOR_TASKS_STATE
+      end
+    end
+  rescue Timeout::Error => error
```

Comment:
rubocop now yells "Naming/RescuedExceptionsVariableName: Use `e` instead of `error`." @ehelms

Reply:
Seriously? uggg

Reply:
ACAB

Reply:
Looks like that cop is designed to help enforce a standard across your project, and defaults to `e`. So we could set it to `error` in the Rubocop config (https://www.rubydoc.info/gems/rubocop/0.75.0/RuboCop/Cop/Naming/RescuedExceptionsVariableName)

Granted, looks like we do not see this error currently because we are using `=> e` everywhere already. Ok - stick with our current standard - file me a Github issue?

Reply:
https://github.com/theforeman/foreman_maintain/issues/885